planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.xml

```diff
@@ -27,7 +27,7 @@
     <include file="$(find-pkg-share obstacle_stop_planner)/launch/obstacle_stop_planner.launch.xml">
       <!-- load config files -->
       <arg name="common_param_path" value="$(var common_param_path)"/>
-      <arg name="adaptive_cruise_control_param_path" value="$(find-pkg-share planning_launch)/config/obstacle_stop_planner/adaptive_cruise_control.param.yaml" />
+      <arg name="obstacle_cruise_control_param_path" value="$(find-pkg-share planning_launch)/config/obstacle_stop_planner/obstacle_cruise_control.param.yaml" />
```

Comment:
Is this right?

Reply:
https://github.com/tier4/autoware_launch/pull/330/commits/fc3fdbd05aa1ed82b2ec0601b4f603d3be76663e
I'm so sorry, I didn't check XML launch file :bow:

Reply:
I confirmed that there are no files including the word "obstacle_cruise_control".